docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1220,7 +1220,77 @@ _____________________
 
 PacketFence supports Eduroam and allows participating institutions to authenticate both locally visiting users from other institutions as well as allowing other institutions to authenticate local users.
 
-In order for PacketFence to allow Eduroam authentication, you need to configure an exclusive Eduroam source in Configuration->Authentication Sources and restart the RADIUS and IPTables services.
+Understanding of the eduroam authentication workflow.
+
+Local authentication
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles